dsgrid/config/common.py

```diff
@@ -23,11 +26,14 @@
         EnergyEndUse,
         EnergyIntensity,
         EnergyIntensityRegression,
-        EnergyServiceEndUse,
+        EnergyServiceDemand,
+        EnergyServiceDemandRegression,
```

Comment:
You probably need to add these to `dsgrid/config/common.py` SUPPORTED_METRIC_TYPES

Reply:
lol, I haven't slept much this week